packages/appstore/src/components/CFDs/cfd-demo-accounts.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { CFD_PLATFORMS } from '@deriv/shared';
+import AccountManager from '../account-manager';
+import { TCFDAccountsProps, TPlatform, TDetailsOfEachMT5Loginid, TStaticAccountProps, TRootStore } from 'Types';
+import { useStores } from 'Stores/index';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+
+const CFDDemoAccounts = ({ isDerivedVisible, isFinancialVisible, current_list }: TCFDAccountsProps) => {
+    const available_demo_accounts: Array<TStaticAccountProps> = [
```

Comment:
```suggestion
    const available_demo_accounts: TStaticAccountProps[] = [
```
More readable this way